### ISSUE TITLE
ANSIBLE0011 all tasks should be named for import_tasks/include_tasks

### ISSUE THREAD

**Issue by abitrolly** [willthames/ansible-lint#299]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible 2.4.0.0
ansible-lint 3.4.15
```

- ansible installation method: OS package
- ansible-lint installation method: OS package

# Desired Behaviour

Is there any rationale why `include_tasks/import_tasks` should be named?

# Actual Behaviour (Bug report only)

```
---
# 
- hosts: websites
  gather_facts: no
  pre_tasks:
  - import_tasks: ansible/tasks_pre_python.yml
```
```
✗ ansible-lint websites.yml
[ANSIBLE0011] All tasks should be named
websites.yml:6
Task/Handler: import_tasks ansible/tasks_pre_python.yml
```

**Comment by blackillzone** [comment]
You need to upgrade to ansible-lint 3.4.17. The support of this new Ansible feature (with import_tasks, import_playbook, include_tasks and include_role) isn't available in your ansible-lint version 3.4.15.

I had the same behavior as you, and upgrading ansible-lint just solved the issue.

If you can't upgrade for any reason, you still can add the skip tag, under your import_tasks, I guess:
```yaml
- import_tasks: ansible/tasks_pre_python.yml
  tags:
    - skip_ansible_lint
```

**Comment by abitrolly** [comment]
Need 3 people to approve update for Fedora https://bodhi.fedoraproject.org/updates/FEDORA-2017-ad58fac94b

**Comment by abitrolly** [comment]
sudo dnf --enablerepo=updates-testing update ansible-lint

That nailed it. I need some more time to test it.

**Comment by abitrolly** [comment]
3.4.17 doesn't complain, but doesn't report absent names in included file.
Anyway, this is now a separate issue - #296
